help.md

```diff
@@ -2,6 +2,7 @@
 Detailed usage information for the various flags of RTLAMR.
 
   - `logfile` writes log statements to the given file. Defaults to `/dev/stdout`.
+  - `logappend` append to, instead of replacing, logfile at startup (only valid when logfile is not /dev/stdout)
```

Comment:
Please revert this change, this file will likely be removed soon because it is out of date.